selinux-playbook.yml

```diff
@@ -4,7 +4,6 @@
   become_method: sudo
   become_user: root
   vars:
-    selinux_policy: targeted
```

Comment:
I'd leave in the example playbook as it is so it's clean that you can override it.

Reply:
Make sense, will change it.